2025-02-05 14:16:44.966 [INF] Test endpoint was called
2025-02-05 14:17:09.940 [INF] Test endpoint was called
2025-02-05 21:04:18.840 [INF] Test endpoint was called
2025-02-05 21:05:35.138 [INF] Test endpoint was called
2025-02-05 21:06:02.290 [INF] This is an information message
2025-02-05 21:06:02.292 [WRN] This is a warning message
2025-02-05 21:06:02.293 [ERR] This is an error message
System.Exception: Test exception
   at Program.<>c.<<<Main>$>b__0_4>d.MoveNext() in /Users/chenhuanqin/Projects/Grab/app/Program.cs:line 47
2025-02-05 23:39:02.280 [INF] Test endpoint was called
2025-02-05 23:39:09.845 [INF] Start processing
2025-02-05 23:42:39.191 [INF] Test endpoint was called
2025-02-05 23:47:26.698 [INF] Test endpoint was called
2025-02-05 23:48:15.219 [INF] 处理已停止
